cmd/fablab/subcmd/exec_loop.go

```diff
@@ -0,0 +1,147 @@
+/*
+	Copyright 2019 NetFoundry Inc.
```

Comment:
I thought we just remove the year?

Reply:
Fixed the copyright messages on all files in fablab